tests/snuba/search/test_backend.py

```diff
@@ -0,0 +1,733 @@
+from __future__ import absolute_import
+
+import pytest
+from datetime import datetime, timedelta
+from django.conf import settings
+
+from sentry.event_manager import ScoreClause
+from sentry.models import (
+    Environment, GroupAssignee, GroupBookmark, GroupStatus, GroupSubscription,
+    Release, ReleaseEnvironment, ReleaseProjectEnvironment
+)
+from sentry.search.base import ANY
+from sentry.search.django.backend import get_latest_release
+from sentry.search.snuba.backend import SnubaSearchBackend
+from sentry.testutils import SnubaTestCase
+
+
+class SnubaSearchTest(SnubaTestCase):
```

Comment:
These are now copied from the Django search tests, so that we test the same stuff.

Some changes have to be made to the "fixture" data, such as anchoring event times to a recent time (vs them being static) since Snuba currently searches 90 days by default.